### PR TITLE
[full-ci][tests-only] revert web pr #6216

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
this pr reverts #6216

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -9,7 +9,7 @@ OC_CI_GOLANG = "owncloudci/golang:1.17"
 OC_CI_NODEJS = "owncloudci/nodejs:14"
 OC_CI_PHP = "owncloudci/php:7.4"
 OC_CI_WAIT_FOR = "owncloudci/wait-for:latest"
-OC_TESTING_MIDDLEWARE = "owncloud/owncloud-test-middleware:1.1.0"
+OC_TESTING_MIDDLEWARE = "owncloud/owncloud-test-middleware:1.0.0"
```
Comment (pascalwengerter) [inline .drone.star:12]: Why are we reverting the middleware version? 🤔

Comment (kiranparajuli589) [reply .drone.star]: fixed

Comment (phil-davis) [reply .drone.star]: we were going to just revert that whole PR, to make sure that we go back to where we were earlier, then tomorrow think about just bumping the middleware version used by the nightwatch acceptance test suite.

Anyway, if CI passes, then great.

**Review by phil-davis — APPROVED** [review]
LGTM, if CI passes. This should remove the middleware from e2e tests.

**Pull request merged**